runtime/common/src/apis.rs

```diff
@@ -481,16 +481,17 @@ macro_rules! impl_runtime_apis_plus_common {
 						// pallet ethereum index: 1
 						// transact call index: 1
 						// Transaction enum variant: 1
-						// chain_id 8 bytes
```

Comment:
This is not right, the type is `u64` which is represented with 8 bytes.

Reply:
it's Option<u64> so we need 1 extra byte for the option